tests/host_callback_test.py

```diff
@@ -2456,6 +2456,8 @@ def helper_check_callback_errors(self, thunk: Callable,
                                    expected_exc_txt: str):
     """Calls thunk() and checks for expected exceptions.
     """
+    if not FLAGS.jax_host_callback_outfeed:
+      raise SkipTest("TODO: implement error handling for customcall")
```

Comment:
Could you create an issue for this and link it in a comment?

Reply:
I will create an issue. But there are 3 "replicas" of this TODO(actually I copied these 2 lines from other testcase), add a comment for each seems too verbose.